src/native/entry.cc

```diff
@@ -127,6 +127,25 @@ static void releaseCreateLock() {
   std::atomic_thread_fence(std::memory_order_release);
 }
 
+bool is_prefix(const char* prefix, char* str)
+{
+  return strstr(str, prefix) == str + sizeof(char);
```

Comment:
I'm not following what's going on here - could you please clarify? Specifically, what's the idea with the ` + sizeof(char)`?

Reply:
This was made to avoid making extra strings starting with `L` to match the class signature. Will clarify it. And it might be clearer to use just `1` instead of `sizeof`.
But I've forgot to remove aforementioned strings itself, and this code will not work. Sorry for that, I will fix it.